pyiso/pjm.py

```diff
@@ -60,11 +81,55 @@ def fetch_edata_series(self, data_type, params=None):
 
         # parse html to df
         dfs = pd.read_html(response.content, header=0, index_col=0, parse_dates=True)
-        df = self.utcify_index(dfs[0])
+        df = self.utcify_index(dfs[0], tz_name='utc')
```

Comment:
Hmm, I have been having problems with the parse_dates function of read_html. thus a lot of the new stuff get's rolled seperately.  Does it work without tz_name?